packages/ui/package.json

```diff
@@ -37,6 +38,7 @@
     "@jest/types": "^27.5.1",
     "@parcel/core": "^2.3.2",
     "@parcel/transformer-sass": "^2.3.2",
+    "@peculiar/webcrypto": "^1.3.3",
```

Comment:
Do we have to install `@peculiar/webcrypto`, since this is running on browser only?

Reply:
ok